src/components/CutoutImage/styled.ts

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components';
+
+export const ImageContainer = styled.div`
+  & > * {
+    width: 400px;
+    height: 400px;
+    object-fit: cover;
+  }
+`;
+
+export const CutoutTopLeft = styled.img`
+  webkit-clip-path: polygon(100% 0, 100% 100%, 0 100%, 0 32%, 35% 32%, 35% 0);
```

Comment:
I think `create-react-app` automatically adds vendor prefixes for us, but I'm not entirely sure